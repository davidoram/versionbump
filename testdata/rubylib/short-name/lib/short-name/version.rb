# ShortLib contains code for the Alchemy AMQP Rack Application server
module ShortLib
  # The current version of ShortLib
  VERSION = '0.5.0'
end%