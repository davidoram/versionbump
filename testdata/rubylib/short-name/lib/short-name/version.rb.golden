# ShortLib contains code for the Alchemy AMQP Rack Application server
module ShortLib
  # The current version of ShortLib
  VERSION = "1.2.3"
end%